src/interfaces/TreeViewProtocol.ts

```diff
@@ -36,6 +36,8 @@ export interface MetalsTreeViewNode {
    *   this node to see the children.
    */
   collapseState?: "collapsed" | "expanded";
+  /** used to contribute item specific actions in the tree */
+  contextValue?: string;
```

Comment:
Same question as in the vs code PR, do we need it only for VS Code or also for other clients?

Reply:
I've removed ContextValue